response_operations_ui/templates/reporting-unit.html

```diff
@@ -66,6 +66,12 @@ <h4 class="neptune">Collection exercises</h4>
 
       <h4 class="neptune">Respondents</h4>
 
+      <div class="ru-survey-enrolment-code u-pb-s">
+        {% if survey.activeIacCode %}
```

Comment:
This probably shouldnt surround the whole div so we dont have unnecessary whitespace